app/src/main/res/xml/video_audio_settings.xml

```diff
@@ -155,5 +164,13 @@
           android:key="@string/seek_duration_key"
           android:summary="%s"
           android:title="@string/seek_duration_title"/>
+
+        <SwitchPreference
+            app:iconSpaceReserved="false"
```

Comment:
Maybe here the [clear_all](https://material.io/resources/icons/?search=clear&icon=clear_all&style=baseline) icon

Reply:
There are no icons in the page. No icon on any page except the main page of settings. Why icons should be in one (or two) of ~30 preference items?

Reply:
Don't put an icon there, so that it remains consistent with the rest of the settings.